include/libtorrent/torrent_handle.hpp

```diff
@@ -1265,6 +1281,24 @@ namespace aux {
 		// client builds.
 		std::shared_ptr<torrent> native_handle() const;
 
+		// Returns true if the torrent is active in its session; returns false if
+		// it has been removed.
+		//
+		// A ``torrent_handle`` remains valid for some time after
```

Comment:
if you don't put types like `torrent_handle` and functions like `session::remove_torrent()` in two quotes ""`", they will turn into links in the documentation, which I think is better.